tests/src/python/test_qgslayertreeview.py

```diff
@@ -37,7 +37,7 @@
 USE_MODEL_TESTER = False
 
 
-if StrictVersion(PYQT_VERSION_STR) >= StrictVersion('5.11'):
+if Version(PYQT_VERSION_STR) >= Version('5.11'):
```

Comment:
Given we require 5.14+ now this logic could be simplified

Reply:
done